### PR TITLE
Added footer component to pages

### PR DESCRIPTION
**Pull request by AlvinCYLo**
## Summary
- Added footer component to all pages
- Added react-router to package.json (seemed to be missing imports from main branch)

## Todos:
- Add styling/color palette to match rest of app when that is finalized

### CODE REVIEW

File: client/src/components/footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    width: "100%",
+  },
+});
+
+export default function Footer() {
+  const classes = useStyles();
+
+  return (
+    <BottomNavigation className={classes.root}>
+      <div>
```
Comment (valarao) [inline client/src/components/footer/Footer.js]: Try using MUI components instead for consistency (div -> Box, a -> Link, etc). 

Reason is MUI can potentially insert theming on the fly if they're already MUI components.

**Review by valarao — CHANGES_REQUESTED** [review]
Left a minor suggestion about how we can set up MUI theming easier - quick fix, but otherwise LGTM

**Review by austincho — APPROVED** [review]
Thanks, Alvin looks good!

**Review by valarao — APPROVED** [review]
LGTM!

**Pull request merged**